src/main/scala/org/clulab/habitus/utils/DefaultContextExtractor.scala

```diff
@@ -16,7 +16,7 @@ class DefaultContextExtractor extends ContextExtractor {
 
       val thisSentMentions = mentionsBySentence(i).distinct
       val (thisSentTBMs, thisSentEvents) = thisSentMentions.partition(_.isInstanceOf[TextBoundMention])
-      val thisSentDates = thisSentTBMs.filter(_.label == "Date")
+      val thisSentDates = thisSentTBMs.filter(m => m.label == "Date" || m.label == "DateRange")
```

Comment:
So even though the label is "DateRange" here, getContext will be called with "Date".  Because of the duplication of the strings and the checking of the string in getContext(), I'll suggest a minor design change next time.  Let's first see whether the code survives.

Reply:
Okay. Sounds good.